### DISCUSSION TITLE
UnityEvent$1 이벤트 타입

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#680] [5. Q&A]
UnityEvent$1 이벤트 타입은
string 이나 다른 interface, class 등은 사용하지 못하나요?
컴파일이 전혀 안되네요?
제약이 있다면 알려주세요.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 현재는 제공하지 못하지만 지원 계획은 내부에서 검토 중에 있는점 참고 부탁 드립니다.
현재 지원하는 부분은 아래 가이드에 있는 내용을 참고하여 주세요.

 https://studio.zepeto.me/guides/properties

감사합니다.

* 참고 이슈 : https://github.com/naverz/zepeto-studio-kor/discussions/573
